### DISCUSSION TITLE
手機對大腦的影響

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1400] [General]
手機在現代生活中已經十分普及，而其實手機正在改變大腦，科學家發現，當閱讀數位媒體，各種網址連結和廣告雜亂無章，以及滾動和滑動手勢都需耗費短期注意力，會導致「高度專注的分心狀態」，即同時處於高度短暫專注，但長時間分心。雖然這種分心看起來是暫時的，但影響長久。科學家表示，頻繁集中的注意力分散，會導致大腦產生持久變化。
2017 年研究發現顯示，只是智慧手機存在，就會降低「可用的認知能力」，因智慧手機讓人隨時取得資訊、娛樂和分散注意力的來源，就算把手機放一邊如桌上或口袋包包，人的認知能力就會下降。

**那我們該怎麼應對呢?**

- 問答式對話：過度依賴手機的人常常很難與人交談或是解釋事情地來龍去脈，與其問「你要用手機做什麼」，不如問「你要用手機玩遊戲嗎」這樣持續對話，不僅讓對方更容易回答，也能建立彼此的信賴關係。

- 設定小目標：突然完全不滑手機、不玩遊戲是不合理的事情，所以將目標訂為「逐步減少使用時間」，像是把現在使用手機的時間減掉一成，達成階段性目標後再減少。或者可以制定「家中使用手機的規則」，例如在客廳充電，不僅可以控制使用使間，也可以在等手機充電的時間促進交流。此外，當成功達到目標，適時獎勵他們也很重要。

- 找到滑手機以外的樂趣：主動創造機會去做滑手機以外的事情，如露營、旅行、烹飪，因為依賴網路、遊戲等虛擬世界，可能在現實生活中與家人親友的聯繫變少，可以試著將家變成舒適交流的場所，讓他們找回人際交流的樂趣。

**Comment by TW-Maverick** [comment]
手機對我們的影響有利也有弊，如何把弊減少，最大化利，是我們要正視的課題！
現在的大學生大部分都住學校附近，生活的範圍除了系館以外，最多的地方就是宿舍或外宿了，那其實3C產品非常容易出現在手邊，不去使用是非常困難的，但可以透過以下方法減少使用時間。

1. IG有可以設定每日時間上限的功能
2. 去圖書館找看看喜歡的書，有閱讀的習慣在現在的社會有越來越少的趨勢
3. 跟同學玩桌遊💯

**Comment by a3240281370** [comment]
手機對大腦的影響有正面和負面兩面。正面方面，手機可以提供便利、即時的資訊，促進學習和思考能力。負面方面，長時間使用手機可能導致注意力分散、減少社交互動和運動，增加焦慮和失眠風險。過度使用手機還可能影響睡眠品質和大腦發育，尤其對兒童和青少年。因此，適度使用手機、定期休息、培養均衡的生活方式是重要的，以維護大腦健康。

**Comment by Alexgns** [comment]
長時間使用手機容易分散注意力，影響專注力和工作效率，且手機螢幕發出的藍光抑制褪黑激素的分泌，干擾睡眠規律，導致睡眠品質下降。
因此要適度使用手機，注意休息和平衡，以維護大腦的健康和功能

**Comment by 1104030360** [comment]
我認為手機對於我們的生活已成為不可或缺的一部分，我們可以使用一些線上的app來控制自己的使用時間，使自己多多接觸現實世界，我也會盡量多走出去運動不坐在位子上玩手機或玩電腦來避免自己對玩手機或玩電腦上癮

**Comment by alstinlin** [comment]
手機雖然提供許多方便的功能，但同時也會讓人無法專心在當下的事情上，所以在使用手機時可以幫自己制定一個規則，比如我自己在讀書時就會把網路關掉，然後把手機放在視線之外的地方，這樣就能避免被通知影響，而分散注意力

**Comment by Chiuhsianglin** [comment]
我認為手機對我的影響很大，現在科技很發達，網路上有很多資訊，甚至是現在的chatGPT，我很常在遇到問題後的第一個想法就去google，想很快地找到解答，而且現在沒帶手機的話就很沒有安全感，會很怕突然有什麼事沒辦法聯絡到別人。

**Comment by Elaine2899** [comment]
我自己是不太依賴手機的人，忙起來甚至會整天不回訊息。我自己對手機的定位是：聯絡親朋好友、存放及查詢資料以及紀錄生活瑣事等較為功能型的作用。手機明顯地影響人與人之間的距離，當然，好的影響在於使人與人之間可以不用碰面就能聯絡感情，但我認為其中也有壞的影響，包括：當人們面對面聚會時，你對面的人還捧著手機不放，形成3C隔閡，這樣的隔閡其實讓我非常不舒服，明明我們已經在咫尺之間，但卻好像隔了數千里這麼遠、話題無法連貫，或是需要仰賴手機裡的內容來維持我們的聊天。所以我覺得過度依賴手機可能會使得人與人之間的距離不如我們想像的"拉遠"了。

**Comment by Tzuhsuannnnn** [comment]
我在網路上有看到一個研究過，林煜軒醫師研究團隊指出每增加使用1小時手機，就會延後入睡時間4.9分鐘，並減少5.5分鐘的睡眠時間，手機使用對睡眠影響是很大的，我十分認同這個，在睡前滑手機往往讓我睡眠時間大大的延後，而睡前沒滑手機往往可以有更好的睡眠品質。

**Comment by Brownnnn78** [comment]
真的不能認同更多，除了無法像以前一樣長時間專注的問題以外，現在加上讀書寫作業打報告都要依靠平板跟筆電，眼睛持續盯著3C產品也是一個需要注意的問題！所以最近開始有努力改成手寫來唸書，有時覺得無紙化是一個很環保的行動，但是想想那些買來卻沒在看用的教科書就沒那麼必須了（誤，要保護眼睛比無紙化更重要！

**Comment by youtinglin** [comment]
不可否認手機對於現代人來說真的是非常重要的，需要利用手機查找資料，與他人聯絡或溝通，如果手機不見或沒電沒網路，就會開始焦慮，怕自己漏掉重要的訊息，導致現代人的社交模式逐漸改變，而且確實容易分心，所以只要有重要的是要做我都會開勿擾模式，把手機丟一邊，讓自己沉浸在當下要做的事，但有時還是會不放心地看，所以我會藉由每個禮拜的一週看手機的時長，去審視自己是否過度用手機，或是有不良的使用習慣，而且手機藍光容易刺激大腦抑制分泌褪黑激素，使睡前看手機容易導致睡眠品質下降。

**Comment by juliaouo** [comment]
手機到底是提供幫助還是造成傷害，完全取決於使用他的人，其實許多工具都是這樣的，如果能夠適當運用，手機能夠幫助人們即時查找資訊並且提供新知以及娛樂。但若是使用不當很容易沉迷進去，造成離不開手機的手機焦慮症。所以重要的是使用者的心態以及自制能力。
